### PR TITLE
Adds new timestamp serialization support

### PR DESCRIPTION
**Pull request by ebenoist**
This supports serializing the timestamp type into a scalar value, which supports current versions of protobuf serialization rules for json

### CODE REVIEW

File: plugin.go [hunk 1]
```diff
@@ -31,6 +35,8 @@ type plugin struct {
 	out *bytes.Buffer
 }
 
+var timestampDefined bool
```
Comment (reubenbrown) [inline plugin.go:38]: I don't think we need this anymore

File: plugin_test.go [hunk 2]
```diff
@@ -8,9 +8,31 @@ import (
...
+	plugin := &plugin{out: &bytes.Buffer{}}
+	res, _ := plugin.Generate(req)
+	content := res.GetFile()[0].GetContent()
+
+	if !strings.Contains(content, "scalar Timestamp") {
+		t.Errorf("Expected the schema to include a scalar definition for Time, %s", content)
+	}
+
+	if !strings.Contains(content, "createdAt: Timestamp") {
+		t.Errorf("Expected the schema to reflect the ISO8601 json serialization format, %s", content)
```
Comment (reubenbrown) [inline plugin_test.go:32]: 🔥

File: plugin.go (hunk 1 above)
Comment (ebenoist) [reply plugin.go:38]: nice catch, thanks!

**Comment by stevenweber** [comment]
This is no longer needed right?

**Comment by ebenoist** [comment]
Ha, it very much is. It's already vendored in prod :/

**Pull request merged**